src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+const Big = require('big.js');
+
+const operate = (ValueOne, ValueTwo, opt) => {
+  let result;
+  if (opt === '+') {
+    result = Big(ValueOne).plus(Big(ValueTwo));
+  }
+
+  if (opt === '+/-') {
+    result = Big(ValueTwo) * (-1);
+  }
+
+  if (opt === '-') {
+    result = Big(ValueOne).minus(Big(ValueTwo));
+  }
+
+  if (opt === 'x') {
+    result = Big(ValueOne).times(Big(ValueTwo));
+  }
+
+  if (opt === '÷') {
+    if (ValueTwo === '0') {
+      result = '0';
+    } else {
+      result = Big(ValueOne).div(ValueTwo);
+    }
+  }
```

Comment:
If `ValueTwo` is ZERO, you should return `NaN` or `Infinity` or `zero division error`
It's not zero......

Reply:
thanks, Shey I will make the change and proceed to merge